sroka/api/athena/athena_api_helpers.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+from botocore.exceptions import ClientError, EndpointConnectionError
+from retrying import retry
+
+
+def return_on_exception(filename):
+    if filename:
```

Comment:
`return filename or pd.DataFrame([])`

Reply:
changing to:
`return None if filename else pd.DataFrame([])`